applications/serializers.py

```diff
@@ -155,6 +156,21 @@ class BootcampApplicationSerializer(serializers.ModelSerializer):
     bootcamp_run = BootcampRunSerializer()
     has_payments = serializers.SerializerMethodField()
     enrollment = serializers.SerializerMethodField()
+    certificate_link = serializers.SerializerMethodField()
+
+    def get_certificate_link(self, application):
+        """Returns certificate link if both certificate and certificate template are presentand enabled"""
+        if getattr(settings, "FEATURE_ENABLE_CERTIFICATES_FOR_USERS", False):
```

Comment:
```suggestion
        if settings.FEATURES.get("ENABLE_CERTIFICATES_FOR_USERS", False):
```

Reply:
Updated to `FEATURE_ENABLE_CERTIFICATE_USER_VIEW` for now.